assets/app/scripts/controllers/catalog/images.js

```diff
@@ -25,7 +25,6 @@ angular.module('openshiftConsole')
               imageRepoTag: imageRepoTag,
               name: imageRepo.metadata.name + ":" + imageRepoTag
             };
-            $scope.images.push(image);
```

Comment:
I don't understand this well enough to know if this is correct... this used to include every tag under `imageRepo.status.tags` in the `other images` list. Now, it only includes those images when `imageRepo.spec.tags` is non-empty. @ncdc @bparees does this sound right to you?